Sub nav

puglisac opened the pull request:
Working on sub-nav.

<img width="1062" alt="Screen Shot 2021-03-02 at 4 01 16 PM" src="https://user-images.githubusercontent.com/66032671/109721201-aafa0e00-7b70-11eb-88cf-dc34911b8a46.png">

puglisac committed:
create sub-nav component. add some styles

puglisac committed:
create sub-nav component. add some styles

puglisac committed:
create sub-nav component. add some styles

puglisac committed:
created todo comment

daemyonc commented:
I modified the height of the bar a bit which you can see in new mocks in the Joblist UI issue.

puglisac committed:
Added sign in with slack button

puglisac commented:
I modified the height and added the sign in with slack button.

<img width="920" alt="Screen Shot 2021-03-03 at 10 43 41 AM" src="https://user-images.githubusercontent.com/66032671/109840054-55277380-7c0d-11eb-8ed9-e13101b081ca.png">

ShaunWMusic commented:
![547F5048-6A9B-4BAE-83DF-F3551140381E](https://user-images.githubusercontent.com/16964213/109841370-98361680-7c0e-11eb-914f-cceeb74a085d.jpeg)

can you line of the jobs right side with this design and also line up the slack button on the right as well?

puglisac committed:
adjusting title and button alignment

puglisac commented:
Yea that looks way nicer I think!

<img width="864" alt="Screen Shot 2021-03-03 at 11 10 32 AM" src="https://user-images.githubusercontent.com/66032671/109843937-4478fc80-7c11-11eb-950a-87040363af0b.png">

ShaunWMusic commented on app/styles/app.css:
```diff
@@ -0,0 +1,59 @@
+    justify-content: start;
+    align-items: center;
+}
+.sub-nav-title{
```
we can change this file to .scss if we want to add variables that are reused

ShaunWMusic commented on app/templates/application.hbs:
```diff
@@ -1 +1,5 @@
\ No newline at end of file
+<TopNav />
+<SubNav />
+{{!-- <main>
```
we can remove the commented out section

ShaunWMusic commented on tests/integration/components/sub-nav/component-test.js:
```diff
@@ -0,0 +1,26 @@
+module('Integration | Component | sub-nav', function(hooks) {
+  setupRenderingTest(hooks);
+
+  test('it renders', async function(assert) {
```
feel free to add tests and try it out

puglisac replied:
Thank you Shaun! I added a couple tests.

ShaunWMusic reviewed (approved):
looking good! once damion merges his, then you can merge this one

puglisac committed:
added tests

puglisac committed:
added tests

puglisac committed:
fixed alignment of dynamic text in sub-nav

daemyonc committed:
Merge branch 'master' into sub-nav
